docs/lib/datastore/fragments/native_common/setup-auth-rules.md

```diff
@@ -0,0 +1,123 @@
+Amplify gives you the ability to limit which individuals or groups should have access to create, read, update, or delete data on your various schema types by specifying an `@auth` directive.
+
+The following is a high level overview of the various authorization scenarios we support in the Amplify libraries.  Each scenario has a variety of options you can tune to fit the needs of your application.
```

Comment:
Shorten:
"Here's a high-level overview of the authorization scenarios we support in Amplify." 
Nit: remove the double whitespace in "libraries.  Each"

Reply:
Shorten: Each scenario has options you can tune to fit the needs of your application.